### PR TITLE
[DOCU-2402] Remove KM and decK, and HTTPie instructions from comprehensive docs.

### PR DESCRIPTION
**Pull request by Guaris**
### Summary
remove KM, Deck, and HTTPIE instructions from comprehensive docs. 
This also includes renaming according to the IA sheet. 
This PR does not contain new KM docs or new decK docs. Those will be added in a later PR. 
This PR does not contain major updates to these docs. That will be added in a later PR. 

### Reason
[[DOCU-2402]](https://konghq.atlassian.net/browse/DOCU-2402)

### Testing
soon

[DOCU-2402]: https://konghq.atlassian.net/browse/DOCU-2402?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/gateway/get-started/comprehensive/improve-performance.md [hunk 1]
```diff
@@ -18,110 +18,16 @@ Use proxy caching so that Upstream services are not bogged down with repeated re
...
-{% navtab Using the Admin API %}
-
 Call the Admin API on port `8001` and configure plugins to enable in-memory caching globally, with a timeout of 30 seconds for Content-Type `application/json`.
 
-<!-- codeblock tabs -->
-{% navtabs codeblock %}
-{% navtab cURL %}
+
 ```sh
 curl -i -X POST http://<admin-hostname>:8001/plugins \
```
Comment (cloudjumpercat) [inline src/gateway/get-started/comprehensive/improve-performance.md:25]:
```suggestion
curl -i -X POST http://localhost:8001/plugins \
```

File: src/gateway/get-started/comprehensive/improve-performance.md [hunk 2]
```diff
@@ -130,20 +36,10 @@ step.
 
 Access the */mock* route using the Admin API and note the response headers:
 
-<!-- codeblock tabs -->
-{% navtabs codeblock %}
-{% navtab cURL %}
+
 ```sh
 curl -i -X GET http://<admin-hostname>:8000/mock/request
```
Comment (cloudjumpercat) [inline src/gateway/get-started/comprehensive/improve-performance.md:41]:
```suggestion
curl -i -X GET http://localhost:8000/mock/request
```

File: src/gateway/get-started/comprehensive/improve-performance.md [hunk 3]
```diff
@@ -175,20 +71,10 @@ X-Kong-Upstream-Latency: 1
 
 To test more rapidly, the cache can be deleted by calling the Admin API:
 
-<!-- codeblock tabs -->
-{% navtabs codeblock %}
-{% navtab cURL %}
+
 ```sh
 curl -i -X DELETE http://<admin-hostname>:8001/proxy-cache
```
Comment (cloudjumpercat) [inline src/gateway/get-started/comprehensive/improve-performance.md:76]:
```suggestion
curl -i -X DELETE http://localhost:8001/proxy-cache
```

File: src/gateway/get-started/comprehensive/load-balancing.md [hunk 4]
```diff
@@ -24,170 +24,37 @@ In the following example, you’ll use an application deployed across two differ
...
-{% endnavtab %}
-{% navtab Using the Admin API %}
-
 Call the Admin API on port `8001` and create an Upstream named `example_upstream`:
 
-<!-- codeblock tabs -->
-{% navtabs codeblock %}
-{% navtab cURL %}
 ```sh
 curl -X POST http://<admin-hostname>:8001/upstreams \
```
Comment (cloudjumpercat) [inline src/gateway/get-started/comprehensive/load-balancing.md:30]:
```suggestion
curl -X POST http://localhost:8001/upstreams \
```

File: src/gateway/get-started/comprehensive/load-balancing.md [hunk 5]
```diff
@@ -24,170 +24,37 @@ In the following example, you’ll use an application deployed across two differ
...
-{% endnavtabs %}
-<!-- end codeblock tabs -->
-
 Update the service you created previously to point to this upstream:
 
-<!-- codeblock tabs -->
-{% navtabs codeblock %}
-{% navtab cURL %}
 ```sh
 curl -X PATCH http://<admin-hostname>:8001/services/example_service \
```
Comment (cloudjumpercat) [inline src/gateway/get-started/comprehensive/load-balancing.md:36]:
```suggestion
curl -X PATCH http://localhost:8001/services/example_service \
```

File: src/gateway/get-started/comprehensive/load-balancing.md [hunk 6]
```diff
@@ -24,170 +24,37 @@ In the following example, you’ll use an application deployed across two differ
...
 
 Add two targets to the upstream, each with port 80: `mockbin.org:80` and
 `httpbin.org:80`:
 
-<!-- codeblock tabs -->
-{% navtabs codeblock %}
-{% navtab cURL %}
+
 ```sh
 curl -X POST http://<admin-hostname>:8001/upstreams/example_upstream/targets \
```
Comment (cloudjumpercat) [inline src/gateway/get-started/comprehensive/load-balancing.md:45]:
```suggestion
curl -X POST http://localhost:8001/upstreams/example_upstream/targets \
```

File: src/gateway/get-started/comprehensive/load-balancing.md [hunk 7]
```diff
@@ -24,170 +24,37 @@ In the following example, you’ll use an application deployed across two differ
...
   --data target='mockbin.org:80'
 
 curl -X POST http://<admin-hostname>:8001/upstreams/example_upstream/targets \
```
Comment (cloudjumpercat) [inline src/gateway/get-started/comprehensive/load-balancing.md:48]:
```suggestion
curl -X POST http://localhost:8001/upstreams/example_upstream/targets \
```

File: src/gateway/get-started/comprehensive/load-balancing.md [hunk 8]
```diff
@@ -24,170 +24,37 @@ In the following example, you’ll use an application deployed across two differ
...
-    deck sync
-    ```
-{% endnavtab %}
-{% endnavtabs %}
 
 You now have an Upstream with two targets, `httpbin.org` and `mockbin.org`, and a service pointing to that Upstream.
 
 ## Validate the Upstream Services
 
 1. With the Upstream configured, validate that it’s working by visiting the route `http://<admin-hostname>:8000/mock` using a web browser or CLI.
```
Comment (cloudjumpercat) [inline src/gateway/get-started/comprehensive/load-balancing.md:56]:
```suggestion
1. With the Upstream configured, validate that it’s working by visiting the route `http://<localhost:8000/mock` using a web browser or CLI.
```

File: src/gateway/get-started/comprehensive/manage-teams.md [hunk 9]
```diff
@@ -37,78 +37,14 @@ In the following sections, you will need the `kong_admin` account’s password t
...
-{% endnavtab %}
-{% navtab Using the Admin API %}
 Create a new workspace called SecureWorkspace, substituting the `kong_admin`
 account’s password in place of `<super-user-token>`:
 
-<!-- codeblock tabs -->
-{% navtabs codeblock %}
-{% navtab cURL %}
 ```sh
 curl -X POST http://<admin-hostname>:8001/workspaces \
```
Comment (cloudjumpercat) [inline src/gateway/get-started/comprehensive/manage-teams.md:44]:
```suggestion
curl -X POST http://localhost:8001/workspaces \
```

File: src/gateway/get-started/comprehensive/manage-teams.md [hunk 10]
```diff
@@ -123,184 +59,60 @@ http :8001/workspaces \
...
-
-    If the registration link has expired, you can generate a new one by logging in with your `kong_admin` administrator and generating a new link.
-
-7. Enter a new password for your new administrator (save this in a secure place) and click on the **Register** button.
-
-    If everything went well, you should see an “Account Setup Success” message.
-{% endnavtab %}
-{% navtab Using the Admin API %}
-
 <div class="alert alert-warning">
```
Comment (cloudjumpercat) [inline src/gateway/get-started/comprehensive/manage-teams.md:66]: Should this note be in Markdown?

File: src/gateway/get-started/comprehensive/manage-teams.md [hunk 11]
```diff
@@ -123,184 +59,60 @@ http :8001/workspaces \
...
 
 Create a new user named `secureworkspaceadmin` with the RBAC token
 `secureadmintoken`:
 
-<!-- codeblock tabs -->
-{% navtabs codeblock %}
-{% navtab cURL %}
+
 ```sh
 curl -X POST http://<admin-hostname>:8001/SecureWorkspace/rbac/users \
```
Comment (cloudjumpercat) [inline src/gateway/get-started/comprehensive/manage-teams.md:75]:
```suggestion
curl -X POST http://localhost:8001/SecureWorkspace/rbac/users \
```

File: src/gateway/get-started/comprehensive/manage-teams.md [hunk 12]
```diff
@@ -123,184 +59,60 @@ http :8001/workspaces \
...
-{% endnavtabs %}
-<!-- end codeblock tabs -->
 
 Create a blank role in the workspace and name it `admin`:
 
-<!-- codeblock tabs -->
-{% navtabs codeblock %}
-{% navtab cURL %}
 ```sh
 curl -X POST http://<admin-hostname>:8001/SecureWorkspace/rbac/roles \
```
Comment (cloudjumpercat) [inline src/gateway/get-started/comprehensive/manage-teams.md:84]:
```suggestion
curl -X POST http://localhost:8001/SecureWorkspace/rbac/roles \
```

File: src/gateway/get-started/comprehensive/manage-teams.md [hunk 13]
```diff
@@ -123,184 +59,60 @@ http :8001/workspaces \
...
-<!-- end codeblock tabs -->
 
 Give the `admin` role permissions to do everything on all endpoints in the
 workspace:
 
-<!-- codeblock tabs -->
-{% navtabs codeblock %}
-{% navtab cURL %}
 ```sh
 curl -X POST http://<admin-hostname>:8001/SecureWorkspace/rbac/roles/admin/endpoints/ \
```
Comment (cloudjumpercat) [inline src/gateway/get-started/comprehensive/manage-teams.md:93]:
```suggestion
curl -X POST http://localhost:8001/SecureWorkspace/rbac/roles/admin/endpoints/ \
```

File: src/gateway/get-started/comprehensive/manage-teams.md [hunk 14]
```diff
@@ -123,184 +59,60 @@ http :8001/workspaces \
...
-{% endnavtabs %}
-<!-- end codeblock tabs -->
 
 Grant the `admin` role to `secureworkspaceadmin`:
 
-<!-- codeblock tabs -->
-{% navtabs codeblock %}
-{% navtab cURL %}
 ```sh
 curl -X POST http://<admin-hostname>:8001/SecureWorkspace/rbac/users/secureworkspaceadmin/roles/ \
```
Comment (cloudjumpercat) [inline src/gateway/get-started/comprehensive/manage-teams.md:103]:
```suggestion
curl -X POST http://localhost:8001/SecureWorkspace/rbac/users/secureworkspaceadmin/roles/ \
```

File: src/gateway/get-started/comprehensive/manage-teams.md [hunk 15]
```diff
@@ -123,184 +59,60 @@ http :8001/workspaces \
...
-
-3. You can also verify that this user’s administration rights are limited. As this user, if you open the Teams tab and try to add new administrators, Admin API users (RBAC users), Groups, or Roles, you won’t have the permissions to do so.
-
-{% endnavtab %}
-{% navtab Using the Admin API %}
 1. Try to access the `default` workspace using `secureworkspaceadmin`'s user token.
 
     *Using cURL:*
     ```sh
     curl -H Kong-Admin-Token:secureadmintoken -X GET http://<admin-hostname>:8001/default/rbac/users
```
Comment (cloudjumpercat) [inline src/gateway/get-started/comprehensive/manage-teams.md:114]:
```suggestion
    curl -H Kong-Admin-Token:secureadmintoken -X GET http://localhost:8001/default/rbac/users
```

File: src/gateway/get-started/comprehensive/manage-teams.md [hunk 16]
```diff
@@ -314,49 +126,11 @@ http :8001/SecureWorkspace/rbac/users/secureworkspaceadmin/roles/ \
     ```sh
     curl -H Kong-Admin-Token:secureadmintoken -X GET http://<admin-hostname>:8001/SecureWorkspace/rbac/users
```
Comment (cloudjumpercat) [inline src/gateway/get-started/comprehensive/manage-teams.md:127]:
```suggestion
    curl -H Kong-Admin-Token:secureadmintoken -X GET http://localhost:8001/SecureWorkspace/rbac/users
```

File: src/gateway/get-started/comprehensive/prepare.md [hunk 17]
```diff
@@ -19,88 +19,17 @@ In this guide, an instance of {{site.base_gateway}} is referenced via
 `<admin-hostname>`. Make sure to replace `<admin-hostname>` with the hostname
```
Comment (cloudjumpercat) [inline src/gateway/get-started/comprehensive/prepare.md:19]: Remove this sentence if we decide to use "localhost" in examples

File: src/gateway/get-started/comprehensive/prepare.md [hunk 18]
```diff
@@ -19,88 +19,17 @@ In this guide, an instance of {{site.base_gateway}} is referenced via
...
 or HTTPie.
 
 View the current configuration by issuing the following command in a terminal
 window:
 
-<!-- codeblock tabs -->
-{% navtabs codeblock %}
-{% navtab cURL %}
 ```bash
 curl -i -X GET http://<admin-hostname>:8001
```
Comment (cloudjumpercat) [inline src/gateway/get-started/comprehensive/prepare.md:31]:
```suggestion
curl -i -X GET http://localhost:8001
```

File: src/gateway/get-started/comprehensive/prepare.md [hunk 19]
```diff
@@ -111,20 +40,10 @@ data planes using the Cluster Status CLI.
 
 Run the following from a control plane:
 
-<!-- codeblock tabs -->
-{% navtabs codeblock %}
-{% navtab cURL %}
 ```bash
 curl -i -X GET http://<admin-hostname>:8001/clustering/data-planes
```
Comment (cloudjumpercat) [inline src/gateway/get-started/comprehensive/prepare.md:44]:
```suggestion
curl -i -X GET http://localhost:8001/clustering/data-planes
```

File: src/gateway/get-started/comprehensive/protect-services.md [hunk 20]
```diff
@@ -15,135 +15,24 @@ Kong's [Rate Limiting plugin](/hub/kong-inc/rate-limiting) lets you restrict how
...
-> **Note:** This section sets up the basic Rate Limiting plugin. If you have a {{site.base_gateway}} instance, see instructions for **Using Kong Manager** to set up Rate Limiting Advanced with sliding window support instead.
 
 Call the Admin API on port `8001` and configure plugins to enable a limit of five (5) requests per minute, stored locally and in-memory, on the node.
 
-<!-- codeblock tabs -->
-{% navtabs codeblock %}
-{% navtab cURL %}
+
 ```sh
 curl -i -X POST http://<admin-hostname>:8001/plugins \
```
Comment (cloudjumpercat) [inline src/gateway/get-started/comprehensive/protect-services.md:26]:
```suggestion
curl -i -X POST http://localhost:8001/plugins \
```

File: src/gateway/get-started/comprehensive/protect-services.md [hunk 21]
```diff
@@ -153,12 +42,17 @@ To validate rate limiting, access the API six (6) times from the CLI to confirm
 curl -i -X GET http://<admin-hostname>:8000/mock/request
```
Comment (cloudjumpercat) [inline src/gateway/get-started/comprehensive/protect-services.md:42]:
```suggestion
curl -i -X GET http://localhost:8000/mock/request
```

File: src/gateway/get-started/comprehensive/protect-services.md [hunk 22]
```diff
@@ -153,12 +42,17 @@ To validate rate limiting, access the API six (6) times from the CLI to confirm
...
 ```
 {% endnavtab %}
-{% navtab HTTPie %}
-```sh
-http :8000/mock/request
-```
+
+{% navtab Web Browser %}
+
+1. Enter `<admin-hostname>:8000/mock` and refresh your browser six times.
```
Comment (cloudjumpercat) [inline src/gateway/get-started/comprehensive/protect-services.md:48]:
```suggestion
1. Enter `localhost:8000/mock` and refresh your browser six times.
```

File: src/gateway/get-started/comprehensive/protect-services.md [hunk 23]
```diff
@@ -153,12 +42,17 @@ To validate rate limiting, access the API six (6) times from the CLI to confirm
...
+    After the 6th request, you’ll receive an error message.
```
Comment (cloudjumpercat) [inline src/gateway/get-started/comprehensive/protect-services.md:49]:
```suggestion
    After the sixth request, you’ll receive an error message.
```

File: src/gateway/get-started/comprehensive/protect-services.md [hunk 24]
```diff
@@ -153,12 +42,17 @@ To validate rate limiting, access the API six (6) times from the CLI to confirm
...
+2. Wait at least 30 seconds and try again.
+    The service will be accessible until the sixth (6th) access attempt within a 30-second window.
```
Comment (cloudjumpercat) [inline src/gateway/get-started/comprehensive/protect-services.md:51]:
```suggestion
    The service will be accessible until the sixth access attempt within a 30-second window.
```

File: src/gateway/get-started/comprehensive/secure-services.md [hunk 25]
```diff
@@ -31,65 +31,23 @@ For more information, see [What is API Gateway Authentication?](https://konghq.c
...
 Call the Admin API on port `8001` and configure plugins to enable key
 authentication. For this example, apply the plugin to the */mock* route you
 created:
 
-<!-- codeblock tabs -->
-{% navtabs codeblock %}
-{% navtab cURL %}
+
 ```sh
 curl -X POST http://<admin-hostname>:8001/routes/mocking/plugins \
```
Comment (cloudjumpercat) [inline src/gateway/get-started/comprehensive/secure-services.md:40]:
```suggestion
curl -X POST http://localhost:8001/routes/mocking/plugins \
```

File: src/gateway/get-started/comprehensive/secure-services.md [hunk 26]
```diff
@@ -31,65 +31,23 @@ For more information, see [What is API Gateway Authentication?](https://konghq.c
...
-{% endnavtabs %}
-<!-- end codeblock tabs -->
 
 Try to access the service again:
 
-<!-- codeblock tabs -->
-{% navtabs codeblock %}
-{% navtab cURL %}
 ```sh
 curl -i http://<admin-hostname>:8000/mock
```
Comment (cloudjumpercat) [inline src/gateway/get-started/comprehensive/secure-services.md:47]:
```suggestion
curl -i http://localhost:8000/mock
```

File: src/gateway/get-started/comprehensive/secure-services.md [hunk 27]
```diff
@@ -105,122 +63,32 @@ Before Kong proxies requests for this route, it needs an API key. For this
...
 To create a consumer, call the Admin API and the consumer’s endpoint.
 The following creates a new consumer called **consumer**:
 
-<!-- codeblock tabs -->
-{% navtabs codeblock %}
-{% navtab cURL %}
+
+
 ```sh
 curl -i -X POST http://<admin-hostname>:8001/consumers/ \
```
Comment (cloudjumpercat) [inline src/gateway/get-started/comprehensive/secure-services.md:76]:
```suggestion
curl -i -X POST http://localhost:8001/consumers/ \
```

File: src/gateway/get-started/comprehensive/secure-services.md [hunk 28]
```diff
@@ -105,122 +63,32 @@ Before Kong proxies requests for this route, it needs an API key. For this
...
 Once provisioned, call the Admin API to provision a key for the consumer
 created above. For this example, set the key to `apikey`.
 
-<!-- codeblock tabs -->
-{% navtabs codeblock %}
-{% navtab cURL %}
+
+
 ```sh
 curl -i -X POST http://<admin-hostname>:8001/consumers/consumer/key-auth \
```
Comment (cloudjumpercat) [inline src/gateway/get-started/comprehensive/secure-services.md:88]:
```suggestion
curl -i -X POST http://localhost:8001/consumers/consumer/key-auth \
```

File: src/gateway/get-started/comprehensive/secure-services.md [hunk 29]
```diff
@@ -241,125 +109,26 @@ HTTP/1.1 201 Created
...
-
-{% endnavtab %}
-{% navtab Using the Admin API %}
 To validate the Key Authentication plugin, access the *mocking* route again, using the header `apikey` with a key value of `apikey`.
 
-<!-- codeblock tabs -->
-{% navtabs codeblock %}
-{% navtab cURL %}
 ```sh
 curl -i http://<admin-hostname>:8000/mock/request \
```
Comment (cloudjumpercat) [inline src/gateway/get-started/comprehensive/secure-services.md:117]:
```suggestion
curl -i http://localhost:8000/mock/request \
```

File: src/gateway/get-started/comprehensive/secure-services.md [hunk 30]
```diff
@@ -241,125 +109,26 @@ HTTP/1.1 201 Created
...
-{% endnavtab %}
-{% navtab Using the Admin API %}
 
 Find the plugin ID and copy it:
 
-<!-- codeblock tabs -->
-{% navtabs codeblock %}
-{% navtab cURL %}
 ```sh
 curl -X GET http://<admin-hostname>:8001/routes/mocking/plugins/
```
Comment (cloudjumpercat) [inline src/gateway/get-started/comprehensive/secure-services.md:129]:
```suggestion
curl -X GET http://localhost:8001/routes/mocking/plugins/
```

File: src/gateway/get-started/comprehensive/secure-services.md [hunk 31]
```diff
@@ -368,44 +137,15 @@ Output:
 
 Disable the plugin:
 
-<!-- codeblock tabs -->
-{% navtabs codeblock %}
-{% navtab cURL %}
 ```sh
 curl -X PATCH http://<admin-hostname>:8001/routes/mocking/plugins/{<plugin-id>} \
```
Comment (cloudjumpercat) [inline src/gateway/get-started/comprehensive/secure-services.md:141]:
```suggestion
curl -X PATCH http://localhost:8001/routes/mocking/plugins/{<plugin-id>} \
```

**Review by cloudjumpercat — CHANGES_REQUESTED** [review]
Left a few comments, let me know if you have questions!

File: src/gateway/get-started/comprehensive/expose-services.md [hunk 32]
```diff
@@ -14,7 +14,7 @@ If you are not following the Getting Started workflow, make sure you have
 ## What are Services and Routes?
 
 **Service** and **Route** objects let you expose your services to clients with
-Kong Gateway. When configuring access to your API, you’ll start by specifying a
+{{site.base_gateway}}. When configuring access to your API, you’ll start by specifying a
 Service. In {{site.base_gateway}}, a Service is an entity representing an external
```
[the remark this answers is not in the document]
Comment (Guaris) [reply src/gateway/get-started/comprehensive/expose-services.md:18]: Later

File: src/gateway/get-started/comprehensive/manage-teams.md (hunk 10 above)
Comment (Guaris) [reply src/gateway/get-started/comprehensive/manage-teams.md:66]: it should, but we can comb through it later when we edit these docs.

**Review by cloudjumpercat — APPROVED** [review]
LGTM!

**Pull request merged**